packages/vuetify/src/services/icons/presets/mdi-svg.ts

```diff
@@ -0,0 +1,32 @@
+import { VuetifyIcons } from 'vuetify/types/services/icons'
```

Comment:
There is no relation between that and mdi (except the paths). But it can be any svg path. Imho the file name should be `svg.ts`.